src/Models/UsersReactions.php

```diff
@@ -38,4 +49,16 @@ public function initialize()
 
         $this->setSource('users_reactions');
     }
+
+
+    /**
+     * afterSave.
+     *
+     * @return void
+     */
+    public function afterSave()
+    {
+        $users = Users::findFirstOrFail($this->users_id);
```

Comment:
@FredPeal why not use the relationship?

Reply:
Because the relationships are load in the afterFetch method